standalone/ansible/roles/monitoring/vars/main.yml

```diff
@@ -4,6 +4,8 @@ node_exporter_port: 9100
 node_exporter_web_listen_address: "0.0.0.0:{{ node_exporter_port }}"
 prometheus_postgres_port: 9187
 redis_exporter_port: 9121
+redis_exporter_user: redis_exporter
+redis_exporter_group: redis_exporter
```

Comment:
What does this change do? Why was it necessary?

Reply:
The prometheus exporter and redis exporter were using the same user (they come with the same default name) and one was trying to change some permissions on the user when it was running a process for the other. This was causing a failure.